### PR TITLE
Feedback from Andrew McMiddlin

### PR DESCRIPTION
**Pull request by ajmcmiddlin**
I've tried to break this up over smaller commits that may include some additional explanation in the commit message. Depending on how you work, it might also make it easier to cherry pick what you want to include.

I don't expect that you'll agree with everything here, so please let me know if you want to discuss anything.

### CODE REVIEW

File: slate/source/index.html.md [hunk 1]
```diff
@@ -424,7 +423,7 @@ This endpoint must be hosted on a website with TLS protection (HTTPS).
 
 ## Client-Initiated Backchannel Authentication (CIBA)
 Client Initiated Backchannel Authentication (CIBA) enables a Data Recipient to
-initiate the authentication of an end-user at a Data Holder by means of out-band
```
Comment (lukepopp) [inline slate/source/index.html.md]: out-band is the correct term see https://openid.net/specs/openid-connect-modrna-client-initiated-backchannel-authentication-1_0.html

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Added
+
+ - Todos for linking directly to relevant part(s) of specifications.
+
+### Fixed
+
+ - Various typographical errors.
+ - Padding of time values in date times per ISO8601.
+ - Made it clearer which values are being referred to in _Hashing value for state and authorisation code_.
+ - Removed implication that `acr` claim is optional.
```
Comment (lukepopp) [inline CHANGELOG.md]: acr is optional under FAPI READ

File: slate/source/index.html.md [hunk 3]
```diff
@@ -43,7 +43,6 @@ The detailed change log for the artifact is available [here](https://github.com/
 -   **JWT:** JSON Web Token
 -   **IPLoA:** Identity Proofing Level of Assurance
 -   **LoA:** Level of Assurance
--   **LoaAs:** Levels of Assurance
```
Comment (lukepopp) [inline slate/source/index.html.md]: LoAs is the preferred acronym https://www.secureidnews.com/news-item/level-of-assurance-changes-for-digital-identity-get-real-in-new-nist-800-63-draft/

File: slate/source/index.html.md [hunk 4]
```diff
@@ -655,23 +672,23 @@ The following OIDC claims will be supported:
     -   `string`
     -   Available in ID Token Only.
     -   Authentication Context Class Reference.
-    -   If passed as an essential claim for the ID Token, the Authorization
-        Server must return an `acr` Claim Value that matches one of the requested
+    -   DR must include the `acr` claim as per
+        [FAPI Part 2 §5.2.3.3]](https://openid.net/specs/openid-financial-api-part-2.html#public-client)
```
Comment (lukepopp) [inline slate/source/index.html.md]: acr is not mandatory under FAPI Read profile https://openid.net/specs/openid-financial-api-part-1.html

File: slate/source/index.html.md [hunk 5]
```diff
@@ -742,17 +759,17 @@ This is covered under [CIBA Authentication flow](#bcAuthorisationEndpoint).
...
 Regardless of the Client Authentication method employed, this endpoint must be
-hosted on a website with **[MTLS]** protection (HTTPS).
+hosted on a website with **[MTLS]** protection.
 
 ## UserInfo Endpoint
 The UserInfo Endpoint is an OAuth 2.0 Protected Resource **[OAUTH2]** hosted by
 the Data Holder that returns Claims about the authenticated End-User **[OIDC]**.
 The Data Recipient sends the UserInfo requests using either the HTTP GET or POST
-method. The Access Token obtained from an OIDC authentication request must be
+method. The Access Token obtained from an OIDC token request must be
```
Comment (lukepopp) [inline slate/source/index.html.md]: authentication is correct not token

File: slate/source/index.html.md (hunk 3 above)
Comment (ajmcmiddlin) [reply slate/source/index.html.md]: OK, so s/LoaAs/LoAs/? It mostly seemed weird to me to include the plural of an initialism that was already there.

**Pull request merged**